packages/deploy-script-support/src/coreProposalBehavior.js

```diff
@@ -76,9 +76,13 @@ export const makeCoreProposalBehavior = ({
     );
 
     // Publish the installations for behavior dependencies.
-    entries(installations || {}).forEach(([key, value]) => {
-      produceInstallations[key].resolve(value);
-    });
+    const installAdmin = E(agoricNamesAdmin).lookupAdmin('installation');
+    await Promise.all(
+      entries(installations || {}).map(([key, value]) => {
+        produceInstallations[key].resolve(value);
```

Comment:
I wonder about the timing of this. I see `scaledPriceAuthority` was moved out of the argument destructuring. Was that because of types or because this mechanism didn't work in time?